gensim/models/hdpmodel.py

```diff
@@ -597,9 +901,44 @@ def __init__(self, dictionary=None, topic_data=None, topic_file=None, style=None
         self.style = style
 
     def print_topics(self, num_topics=10, num_words=10):
+        """Gives the most probable `num_words` words from `num_topics` topics.
+
+        Parameters
+        ----------
+        num_topics : int, optional
+            Top `num_topics` to be printed.
+        num_words : int, optional
+            Top `num_words` most probable words to be printed from each topic.
+
+        Returns
+        -------
+        list of tuple of (unicode,numpy.float64) or list of str
+            Output format for `num_words` words from `num_topics` topics depends on the value of `self.style` attribute.
+
+        """
         return self.show_topics(num_topics, num_words, True)
 
     def show_topics(self, num_topics=10, num_words=10, log=False, formatted=True):
+        """Gives the most probable `num_words` words from `num_topics` topics.
```

Comment:
`Give`, `Print` instead of `Gives`, `Prints` in the first line of docstring (here and everywhere).